### DISCUSSION TITLE
Nueva página Maestro de Descriptores

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#48] [General]
Nueva página de Maestro de descriptores, que pasa a hacer más un maestro que un listado de campos, mejorando la creación, edición y administración de los descriptores mejorando la experiencia de usuario.

**Comment by NestorFGG** [comment]
Nueva página con implementación de Diseño

**Comment by aleja550** [comment]
Nunca se realizo ni tampoco está en planeación del producto actual.
